src/qt/transactionrecord.cpp

```diff
@@ -175,42 +175,31 @@ void TransactionRecord::updateStatus(const interfaces::WalletTxStatus& wtx, cons
 
     // For generated transactions, determine maturity
     if (type == TransactionRecord::Generated) {
-        if (wtx.blocks_to_maturity > 0)
-        {
+        if (wtx.blocks_to_maturity > 0) {
```

Comment:
These lines shouldn't be reformatted for no reason

Reply:
I'm changing this function, which seems like the right time for it. Per developer guidelines:

> When writing patches, favor the new style over attempting to mimic the surrounding style